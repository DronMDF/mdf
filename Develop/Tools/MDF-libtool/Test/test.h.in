//
// Copyright (c) 2000-2006 Andrey Valyaev (dron@infosec.ru)
// All rights reserved.
//

#pragma once(__TEST_H__)

extern int errno;

#define NULL ((void *)0)

#pragma prototypes
