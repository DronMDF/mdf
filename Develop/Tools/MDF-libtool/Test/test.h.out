//
// Copyright (c) 2000-2006 Andrey Valyaev (dron@infosec.ru)
// All rights reserved.
//

#ifndef __TEST_H__
#define __TEST_H__

extern int errno;

#define NULL ((void *)0)

#ifdef __cplusplus
extern "C" {
#endif
#ifdef __cplusplus
}
#endif

#endif // __TEST_H__
